.drone.yml

```diff
@@ -41,22 +41,25 @@ steps:
     image: golang:1.13
     commands:
       - make e2e-test
-    volumes:
-      - name: deps
-        path: /go
     environment:
-      MCLI_PUBLIC_API_KEY:
-        from_secret: mcli_public_api_key
-      MCLI_PRIVATE_API_KEY:
-        from_secret: mcli_private_api_key
       MCLI_OPS_MANAGER_URL:
         from_secret: mcli_ops_manager_url
-      MCLI_SERVICE:
-        from_secret: mcli_service
       MCLI_ORG_ID:
         from_secret: mcli_org_id
+      MCLI_PRIVATE_API_KEY:
+        from_secret: mcli_private_api_key
       MCLI_PROJECT_ID:
         from_secret: mcli_project_id
+      MCLI_PUBLIC_API_KEY:
+        from_secret: mcli_public_api_key
+      MCLI_SERVICE:
+        from_secret: mcli_service
+    volumes:
+      - name: deps
+        path: /go
+    depends_on:
```

Comment:
this is the main change here, this enables to run test and lint on parallel, the rest is just drone fmt doing its thing